<!-- SDU Lidar 3d demo launch file -->

<launch>
	<node pkg="rosbag" name="robot" type="play" args="--rate=1 $(find sdu_lidar3d_demo)/bagfiles/2013-11-06-11-00-23.bag" output="screen">
		<remap from="/rosbag/clock" to="/clock"/>
	</node>
	
	<node type="laser_scan_assembler" pkg="laser_assembler" name="sick_lms111_assembler">
		<remap from="scan" to="/laser_lms111"/>
		<param name="max_scans" type="int" value="400" />
		<param name="fixed_frame" type="string" value="base_link" />
	</node>

	<node type="periodic_snapshotter" pkg="laser_assembler" name="periodic_snapshots">
	</node>

	<node pkg="sdu_lidar3d_demo" name="TriangleDrawer" type="TriangleDrawer" output="screen">
		<remap from="/scan" to="/laser_lms111"/>
		<param name="maxDist" type="double" value="2" />
		<param name="minDist" type="double" value="0.01" />
	</node>

	<node pkg="rviz" type="rviz" name="visualization" args="-d $(find sdu_lidar3d_demo)/conf/default.rviz"/>

</launch>

